__tests__/integration/mirador/plugins.html

```diff
@@ -12,89 +12,103 @@
     <script src="../../../node_modules/react-dom/umd/react-dom.development.js"></script>
     <script>document.write("<script type='text/javascript' src='../../../dist/mirador.min.js?v=" + Date.now() + "'><\/script>");</script>
     <script type="text/javascript">
-      class MiradorShareButton extends React.Component {
```

Comment:
This is probably just an artifact of the repo transfer, but do you mind updating the whitespace changes here so that this isn't entirely rewritten?